autometa/common/external/bowtie.py

```diff
@@ -147,6 +148,10 @@ def main(args):
         nproc=args.nproc,
         kwargs=args.kwargs)
 
+
+
+
+
```

Comment:
Is there a reason to insert these blank lines?

Reply:
No, fixed now